neo/rawio/baserawio.py

```diff
@@ -467,24 +478,43 @@ def _get_channel_indexes(self, stream_index, channel_indexes, channel_names, cha
             channel_indexes = self.channel_id_to_index(stream_index, channel_ids)
         return channel_indexes
 
-    def _get_stream_index(self, stream_index):
-        if stream_index is None:
+    def _get_stream_index_from_arg(self, stream_index_arg):
```

Comment:
This change raise make bug!! Have a look to travis and circleci.
Either fix it or back to previous name.

Reply:
Pushed a change to address most of these. From what I see now in the circlci output there are 2 errors remaining in nixio. I don't see those being related to this recent change.

Reply:
Circle ci fails are already foxed in an otehr branch.
I will merge this other branch first and then this one.